pytorch_lightning/__init__.py

```diff
@@ -12,7 +12,6 @@
 # if root logger has handlers, propagate messages up and let root logger process them
 if not _root_logger.hasHandlers():
     _logger.addHandler(logging.StreamHandler())
-    _logger.propagate = False
```

Comment:
The docs say the default is True: https://docs.python.org/3/library/logging.html#logging.Logger.propagate
Didn't we set it to False for a reason? I'm afraid this could break logging configuration we have previously setup. I'm not sure.

Reply:
#6275
The issue was a duplication of all log lines.

Reply:
Yep yep, found a different workaround. Sending this patch specifically in https://github.com/PyTorchLightning/pytorch-lightning/pull/10577